aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -289,7 +289,7 @@ void printTensorNDArray(const Tensor& t) {
   } else {
     if (!mpsShape) {
       mpsShape = getMPSShape(_tensor);
-  }
+    }
```

Comment:
Fix this.

Reply:
This is purposely an indentation fix for `if (!mpsShape) { ` block.